doctrine/event_listeners_subscribers.rst

```diff
@@ -186,12 +186,17 @@ interface and have an event method for each event it subscribes to::
 
 For a full reference, see chapter `The Event System`_ in the Doctrine documentation.
 
-Performance considerations
---------------------------------
+Performance Considerations
+--------------------------
 
-One subtle difference between listeners and subscribers is that Symfony will load
-entity listeners lazily by default as of Symfony 4.2. This means that your listener class will only be fetched
-from the service container (and thus be instantiated) once the event it is linked
-to actually fires.
+One important difference between listeners and subscribers is that Symfony loads
+entity listeners lazily. This means that the listener classes are only fetched
+from the service container (and instantiated) if the related event is actually
+fired.
 
-So whenever possible it is preferable to use entity listeners instead of subscribers.
+That's why it is preferable to use entity listeners instead of subscribers
+whenever possible.
+
+.. versionadded:: 4.2
+    The default lazy behavior of Doctrine entity listeners was introduced in
```

Comment:
technically this is not correct. It was possible even before Symfony 4.2 but you had to manually add a `lazy=true` attribute for the tag. Now since 4.2 it will be always lazy :blush:

Reply:
Ah sorry I missed the "default" while reading it. All good :+1:

Reply:
In that phrase I tried to say: "in 4.2 the new thing is that is lazy by default". But it looks like I failed.

Reply:
@dmaicher in 4.2 they are lazy by default ... or are they mandatory lazy? In other words, can you say `lazy:false` in 4.2?

Reply:
No they are always lazy now. So mandatory.